constants/constants.go

```diff
@@ -142,6 +143,8 @@ ETCD_STRICT_RECONFIG_CHECK=true
 
 	EtcdctlEnvFileTemplate = `export ETCDCTL_API=3
 
+export ETCDCTL_ENDPOINTS={{ .ListenClientURLs.String }}
```

Comment:
Interesting change - it looks right, but I'm wondering how we were passing ListenClientURLs previously?